docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc

```diff
@@ -0,0 +1,250 @@
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.15.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.15.0 relnotes
+
+[[release-notes-8.15.0]]
+== {fleet} and {agent} 8.15.0
+
+Review important information about the {fleet} and {agent} 8.15.0 release.
+
+[discrete]
+[[security-updates-8.15.0]]
+=== Security updates
+
+{fleet-server}::
+* Update {fleet-server} Go version to 1.22.5. {fleet-server-pull}3681[#3681]
+
+[discrete]
+[[new-features-8.15.0]]
+=== New features
+
+The 8.15.0 release Added the following new and notable features.
+
+{fleet-server}::
+* When {fleet-server} runs in `elastic-agent` mode, it's now able to use the enrollment configuration options in `output.elasticsearch.bootstrap` from its policy, instead of overwriting the matching keys in `output.elasticsearch`. {fleet-server-pull}3506[#3506] {fleet-server-issue}3464[#3464]
+* As part of making {fleet} space aware, {fleet-server} now adds a `namespaces` property to created `.fleet-*` documents. {fleet-server-pull}3535[#3535] {fleet-server-issue}3505[#3505]
+
+{agent}::
+* Enable {agent} to monitor and report usage metrics for {elastic-endpoint}. {agent-pull}4789[#4789] {agent-issue}4083[#4083]
+* Add the AWS Asset Inventory input to Cloudbeat. {agent-pull}4804[#4804]
+* Unhide the `--unprivileged` option for the `elastic-agent install` command and mark the usage of the flag as being in a `beta` technical preview state. {agent-pull}4914[#4914]
+* To ensure that {agent} starts correctly when run in a container, ensure that the `statePath` set by the container command generates a Unix socket path that is smaller than 108 characters. {agent-pull}4909[#4909]
+* Enable {agent} to receive an event logger configuration through {fleet}. {agent-pull}4932[#4932] {agent-issue}4874[#4874]
+
+[discrete]
+[[enhancements-8.15.0]]
+=== Enhancements
+
+{fleet}::
+* Use API key for standalone agent onboarding. ({kibana-pull}187133[#187133])
+* Add action for upgrading all agents on a policy. ({kibana-pull}186827[#186827])
+* Change agent policies in edit package policy page. ({kibana-pull}186084[#186084])
+* Create shared package policy. ({kibana-pull}185916[#185916])
+* Make {fleet} & Integrations layouts full width. ({kibana-pull}186056[#186056])
+* Add support for setting `add_fields` processors on all agents under an agent policy. ({kibana-pull}184693[#184693])
+* Introduce `policy_ids` in package policy SO ({kibana-pull}184636[#184636])
```

Comment:
Same here re: reusable policies

Reply:
Same here. Fixed in [82af9e6](https://github.com/elastic/ingest-docs/pull/1201/commits/82af9e649be62b17a2bcef2f8e1d955f38e408cf).